packages/strapi-plugin-users-permissions/admin/src/components/PopUpForm/index.js

```diff
@@ -153,7 +155,9 @@ class PopUpForm extends React.Component {
               key={value}
               label={{
                 id: `users-permissions.PopUpForm.Providers.${
-                  includes(value, 'callback') || includes(value, 'redirect_uri')
+                  includes(value, 'callback') ||
+                  includes(value, 'redirect_uri') ||
+                  includes(value, 'authorize_url')
```

Comment:
You can remove authorize_url here I think

Reply:
Hm, that was added because it appeared to solve this [label problem](https://github.com/strapi/strapi/pull/1616#issuecomment-411782099) from the last PR, but it looks like it's not needed. I'll remove it!